build.fsx

```diff
@@ -191,7 +213,10 @@ Target "Default" DoNothing
 "Default"
   ==> "PrepareRelease"
   ==> "Build.Net35"
+  =?> ("Build.NetCore", isDotnetSDKInstalled)
+  =?> ("RunTests.NetCore", isDotnetSDKInstalled)
   ==> "NuGet"
+  =?> ("Nuget.AddNetCore", isDotnetSDKInstalled)

```

Comment:
This target should probably fail if CoreCLR is not installed, otherwise we risk pushing bad packages from machines that may lack it.